test/ao/sparsity/test_pruner.py

```diff
@@ -11,7 +11,7 @@
 
 logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', level=logging.INFO)
 
-class Model(nn.Module):
+class Linear(nn.Module):
```

Comment:
nit: using the same name as nn module (since we have nn.Linear) might be a bit confusing, maybe we can have a different name like LinearA